packages/fluent-theme/src/components/preChatActivity/PreChatMessageActivity.tsx

```diff
@@ -10,29 +10,40 @@ import StarterPromptsCardAction from './StarterPromptsCardAction.js';
 
 type Props = Readonly<{ activity: WebChatActivity & { type: 'message' } }>;
 
-const { useRenderMarkdownAsHTML } = hooks;
+const { useLocalizer, useRenderMarkdownAsHTML } = hooks;
 
 const PreChatMessageActivity = ({ activity }: Props) => {
   const classNames = useStyles(styles);
   const renderMarkdownAsHTML = useRenderMarkdownAsHTML();
-
-  const html = useMemo(
-    () => (renderMarkdownAsHTML ? { __html: renderMarkdownAsHTML(activity.text || '') } : { __html: '' }),
-    [activity.text, renderMarkdownAsHTML]
-  );
+  const localize = useLocalizer();
 
   const entity = getOrgSchemaMessage(activity?.entities || []);
   const isPlaceHolder = entity?.keywords?.includes('PreChatMessage') && entity.creativeWorkStatus === 'Placeholder';
+  const author = typeof entity?.author !== 'string' ? entity?.author : undefined;
```

Comment:
Maybe if `typeof author === 'string'`, we treat it as `{ name: author, type: 'Person' }` then call it done?

Reply:
Good idea. Making changes